travis/travis_install_nightly

```diff
@@ -138,6 +138,14 @@ else
     # Install websocket-client, required to run JS tests
     pip install websocket-client
 fi
+# The new syntax of JS tests requires the latest tools to run
+if [ ${VERSION%.*} -ge 13 ]; then
```

Comment:
I don't find this necessary. The only reason we're doing this here and not on the big image is that the big image can't be updated anymore without fixing first a bunch of issues. So, I consider this should be always performed, independently of the version, as we would do when updating the big image (the new image is on the way, BTW)

Reply:
Done